db/migrations/0011.add_bluejeans_table.py

```diff
@@ -0,0 +1,42 @@
+#
+# file: migrations/0011.add_bluejeans_table.py
+#
+from yoyo import step
+
+__depends__ = {'0010.add_fkc_course_usage_table'}
```

Comment:
The latest migration file from the master is `011.add_course_sis_ids.py` and you migration should be based on that
https://github.com/tl-its-umich-edu/course-inventory/blob/master/db/migrations/0011.add_course_sis_ids.py

Also, change the numbering on this file to 0012 something like that